pysrc/bytewax/inp.py

```diff
@@ -44,7 +45,7 @@ def tumbling_epoch(
     Yields: Tuples of (epoch, item).
     """
     epoch = 0
-    last_epoch_start_sec = time.time() - epoch_length_sec
```

Comment:
Intentional, but I guess I shouldn't have thrown it in here. This makes the epoch 0 window start with the first item, rather than close with the first item.